src/main/java/TaskList.java

```diff
@@ -0,0 +1,133 @@
+import java.util.ArrayList;
+
+public class TaskList {
+
+    private ArrayList<Task> taskList = new ArrayList<Task>();
+
+    /**
+     * gets details for all available task when the list command is invoked.
+     */
+    public String getTaskList() {
```

Comment:
this is a misleading method name for what it does.
Consider @Override toString() instead.